internal/engine/watchmanager_test.go

```diff
@@ -78,7 +78,6 @@ func TestWatchManager_WatchesReappliedOnDockerComposeMountsChange(t *testing.T)
 	f.SetManifestTarget(target)
 
 	f.ChangeFile(t, "bar")
-	f.ChangeFile(t, "stop")
```

Comment:
why did this test change?

Reply:
When I was writing the new test, I thought it was silly and error-prone that I had to `f.ChangeFile(t, "stop")` myself, and that it'd be much safer to have `f.Stop` do that.

It turned out that `f.Stop` already does that, so I removed the redundant `f.ChangeFile(t, "stop")`s from other tests.